base/operators.jl

```diff
@@ -542,6 +542,8 @@ for op in (:+, :*, :&, :|, :xor, :min, :max, :kron)
     end
 end
 
+function kron! end
+
```

Comment:
this is because `kron` is defined in `Base` (https://github.com/JuliaLang/julia/blob/master/base/operators.jl#L533), thus all other stdlibs need to `import Base: kron`, I think it'd be more consistent to define the generic function in `Base` instead of `LinearAlgebra`.